WebExample/playwright.config.ts

```diff
@@ -0,0 +1,29 @@
+import {defineConfig, devices} from '@playwright/test';
+import * as TEST_CONST from '../testConstants';
+
+export default defineConfig({
+  testDir: './tests',
+  preserveOutput: 'never',
+  outputDir: undefined,
+  webServer: {
+    command: 'yarn web',
+    url: TEST_CONST.LOCAL_URL,
+    reuseExistingServer: !process.env.CI,
+    stdout: 'pipe',
+    stderr: 'pipe',
+  },
+  projects: [
+    {
+      name: 'Chromium',
+      use: {...devices['Desktop Chrome']},
```

Comment:
Why not just `devices['Desktop Chrome']`?

Reply:
this is directly from the documentation
https://playwright.dev/docs/test-projects